.vscode/settings.json

```diff
@@ -0,0 +1,56 @@
+{
+  "[cpp]": {
+    "editor.formatOnSave": false
+  },
+  "[c]": {
+    "editor.formatOnSave": false
+  },
+  "[xml]": {
+    "editor.formatOnSave": false
+  },
+  "files.trimTrailingWhitespace": false,
+  "files.insertFinalNewline": false,
+  "files.associations": {
+    "*.tpl": "html",
+    "*.svetle": "html",
+    "*.mustache": "html",
+    "*.js": "javascript",
+    "*.ts": "typescript",
+    "*.toml": "properties",
+    "*.service": "properties",
+    "*.prettierrc": "json",
+    "*.qc": "c",
```

Comment:
We don't have any TypeScript, JavaScript or SQL and etc...
Is it needed here for some reason?